src/pages/ReportDetailsPage.tsx

```diff
@@ -165,7 +165,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
         // 2. MoneyReport case
         if (caseID === CASES.MONEY_REPORT) {
             if (!reportActions || !transactionThreadReport?.parentReportActionID) {
-                return null;
+                return undefined;
```

Comment:
Ah why are we returning `undefined` instead of `null`?

Reply:
@marcochavezf because `PromotedActions.hold` accepts `reportAction` param with type `ReportAction | undefined`, return `undefined` to avoid ts error.

Reply:
Oh I see, thanks for the clarification!